release-notes/5.0/api-diff/netcoreapp3.1/.NET/standalone/5.0-standalone_System.Collections.Immutable.md

```diff
@@ -0,0 +1,1004 @@
+# System.Collections.Immutable
+
+``` diff
+ {
```

Comment:
@Anipik should you delete these files since you removed them from the TOC?

Reply:
TOC ?

Reply:
Table of contents, this file: https://github.com/dotnet/core/pull/5610/files#diff-2226bc7a0ac5271249dcb658c1c9679c1a13e8e0a4e9c04381cdab801784dbb2

Reply:
yeah i will check whether these files needs to be removed or we need the entry back in TOC